src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -588,52 +611,76 @@ private static unsafe void Ssse3Decode(ref byte* srcBytes, ref byte* destBytes,
             do
             {
                 AssertRead<Vector128<sbyte>>(src, srcStart, sourceLength);
-                Vector128<sbyte> str = Sse2.LoadVector128(src).AsSByte();
+                Vector128<byte> str = Vector128.LoadUnsafe(ref *src);
```

Comment:
Possibly not worth handling in this PR, but we should probably avoid pinning and just take in `ref src`. We can return the total offset for the trailing loop can still be handled (this is currently a double indirection each iteration which isn't great).